File: MaticulaUniversitaria/front/laca/src/app/logic-components/classes/Estudiante.ts
export class Estudiante {
  cedula: number;
  nombre: string;
  apellido: string;
  correo_electronico: string;
  contrasena: string;


  constructor(cedula: number = 0, nombre: string = '', apellido: string = '', correo_electronico: string = '', contrasena: string = '') {
    this.cedula = cedula;
    this.nombre = nombre;
    this.apellido = apellido;
    this.correo_electronico = correo_electronico;
    this.contrasena = contrasena;
  }

}

File: MaticulaUniversitaria/api/project/src/main/java/com/laca/entity/Estudiante.java
package com.laca.entity;

import java.util.Date;

public class Estudiante {
    private int cedula;
    private String nombre;
    private String apellido;
    private String correo_electronico;
    private String contrasena;

    public Estudiante() {
    }

    public Estudiante(int cedula, String nombre, String apellido, String correo_electronico, String contrasena) {
        this.cedula = cedula;
        this.nombre = nombre;
        this.apellido = apellido;
        this.correo_electronico = correo_electronico;
        this.contrasena = contrasena;
    }

    public int getCedula() {
        return cedula;
    }

    public void setCedula(int cedula) {
        this.cedula = cedula;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public String getCorreo_electronico() {
        return correo_electronico;
    }

    public void setCorreo_electronico(String correo_electronico) {
        this.correo_electronico = correo_electronico;
    }

    public String getContrasena() {
        return contrasena;
    }

    public void setContrasena(String contrasena) {
        this.contrasena = contrasena;
    }
}
